python-phoenixdb/phoenixdb/cursor.py

```diff
@@ -27,7 +27,7 @@
 # TODO see note in Cursor.rowcount()
 MAX_INT = 2 ** 64 - 1
 
-ColumnDescription = collections.namedtuple('ColumnDescription', 'name type_code display_size internal_size precision scale null_ok')
+ColumnDescription = collections.namedtuple('ColumnDescription', 'label type_code display_size internal_size precision scale null_ok')
```

Comment:
Nit: While the names are not part of the PEP249 API, we use the names defined in the standard as keys.
Please change this back to name.
(PEP249 has no concept of name vs label, the fact that JDBC driver calls the visible name "label" does not need to be visible here.)